.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 name: CI tests
 
 on:
+  schedule:
+    - cron: '0 7 * * *' # Runs daily at 2:00 AM EST
```

Comment:
For a moment I was thinking whether we should use a different time than for the [CF checks](https://github.com/eisop/checker-framework/blob/82d9275ac58caa1fd9183fad2cf3be278fb0d4d7/.github/workflows/ci.yml#L5), but I guess GitHub will schedule this how it wants.